sample-apps/monaibundle/main.py

```diff
@@ -30,6 +34,15 @@
 class MyApp(MONAILabelApp):
     def __init__(self, app_dir, studies, conf):
         self.models = get_bundle_models(app_dir, conf)
+        # Add Epistemic model for scoring
+        self.epistemic_models = (
+            get_bundle_models(app_dir, conf, conf_key="epistemic_model") if conf.get("epistemic_model") else None
+        )
+        if self.epistemic_models:
+            # Get epistemic parameters
+            self.epistemic_max_samples = int(conf.get("epistemic_max_samples", "0"))
+            self.epistemic_simulation_size = int(conf.get("epistemic_simulation_size", "5"))
+            self.scoring_dropout = float(conf.get("scoring_dropout", "0.2"))
```

Comment:
let this also start as `epistemic_dropout` 
And we need to mention these as part of readme.. something like this..
https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/monaibundle#additional-configs

Reply:
Thanks for suggestions! changed to epistemic_dropout. Added configs to readme.